public/components/datasources/components/manage/data_connection.tsx

```diff
@@ -20,22 +20,34 @@ import {
   EuiTitle,
 } from '@elastic/eui';
 import React, { useEffect, useState } from 'react';
+import {
+  useLoadAccelerationsToCache,
+  useLoadDatabasesToCache,
+  useLoadTablesToCache,
+} from '../../../../../public/framework/catalog_cache/cache_loader';
+import { DirectQueryLoadingStatus } from '../../../../../common/types/explorer';
+import { CatalogCacheManager } from '../../../../../public/framework/catalog_cache/cache_manager';
 import {
   DATACONNECTIONS_BASE,
   observabilityIntegrationsID,
   observabilityLogsID,
   observabilityMetricsID,
 } from '../../../../../common/constants/shared';
-import { DatasourceType } from '../../../../../common/types/data_connections';
 import { coreRefs } from '../../../../framework/core_refs';
 import { getRenderCreateAccelerationFlyout } from '../../../../plugin';
 import { NoAccess } from '../no_access';
+import {
+  CachedAcceleration,
+  CachedDatabase,
+  CachedTable,
+  DatasourceType,
+} from '../../../../../common/types/data_connections';
+import { AssociatedObjectsTab } from './associated_objects/associated_objects_tab';
 import { AccelerationTable } from './accelerations/acceleration_table';
 import { AccessControlTab } from './access_control_tab';
-import { AssociatedObjectsTab } from './associated_objects/associated_objects_tab';
 import { mockAssociatedObjects } from './associated_objects/utils/associated_objects_tab_utils';
 
-interface DatasourceDetails {
+export interface DatasourceDetails {
```

Comment:
If exporting should be moved to `common/types/` in root folder.

Reply:
Do we still need this here?

Reply:
Moving that out in next commit.